teacher.rb

```diff
@@ -0,0 +1,15 @@
+require './person'
+
+# the teacher class
+class Teacher < Person
+  attr_reader :specialization
+
+  def initialize(age, specialization, name: 'Unkown', parent_permission: true)
+    super(name, age, parent_permission)
```

Comment:
- [ ] In your initialize function coming from the Person class, the first input argument is age not name, please kindly make this important change to your `super` method in your codebase :+1: